src/assets/i18n/en.json5

```diff
@@ -1516,6 +1516,7 @@
 
   "forgot-password.form.submit": "Submit password",
 
+  "forgot-password.form.error.robust-password": "Selected password is invalid, it should be at least 8 and at most 15 characters long and must contain at least one uppercase and one lowercase character and a number.",
```

Comment:
In my opinion, as first step, we could share the same i18n key in all components, without having it redundant in many places. 
On the other hand, having it in 422 message would leave responsibility of defining minimum requirements on backend, so in case of changes on minimum requirement, we would have to modify both validation regex and error message just on backend. I think this could be a future improvement.